recipes/sardana-limaccd/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "sardana-limaccd" %}
+{% set version = "0.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ALBA-Synchrotron/sardana-limaccd/archive/{{ version }}.tar.gz
+  sha256: 238d3dc7ffd228279dda8e2b31ae373b58488f97f407643f68fcfe7ddc58178c
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - sardana
+
+test:
+  imports:
+    - sardana_limaccd
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ALBA-Synchrotron/sardana-limaccd
+  summary: LimaCCDs Tango plugins for Sardana
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - rhomspuron
+    - ovallcorba
```

Comment:
Need approval from the second maintainer to be listed

Reply:
Hi @xhochy , I just approved the reviewed changes, is this what was missing?
Thanks!